index.js

```diff
@@ -0,0 +1,39 @@
+import {
+  // eslint-disable-next-line max-len
+  inputTitle, inputAuthor, btn, listLink, addLink, contactLink, booksSection, addBookSection, contactSection,
```

Comment:
- [OPTIONAL] Please break this line of code into two lines of code. This will fix the problem without having to disable linters. 🤓

Reply:
Thanks for the feedback @GiuseppeTG 😊